pixano/data/settings.py

```diff
@@ -13,15 +13,58 @@
 
 
 from pathlib import Path
+from s3path import S3Path, register_configuration_parameter
+import boto3
+from urllib.parse import urlparse
 
 from pydantic_settings import BaseSettings
+from typing import Optional
 
 
 class Settings(BaseSettings):
     """Dataset library settings
 
     Attributes:
-        data_dir (Path): Dataset library directory
+        data_dir (Path): Dataset library directory, or url to S3 path (eg. "s3://<bucket>[/<key>]")
+
+        If data_dir is a S3 url, the following environment variables can be provided
+            aws_endpoint (str (Optional)): <S3 compatible storage url>. Use AWS if not provided
+            aws_access_key_id (str (Optional)): <access key>
+            aws_secret_access_key (str (Optional)): <secret key>
+            aws_region (str (Optional)): <region> Not always required for private S3 storages
+            local_model_dir (str (Optional)): Path to model (local) if not on S3 storage (in data_dir / models)
     """
```

Comment:
To be processed correctly for the documentation website, S3 attributes should be on the same indentation level as data_dir, without the text above them.

The type syntax is (str, optional) instead of (str (Optional))

The "<>" quotes should be removed from the attributes descriptions.